src/govpress-unit/code-review.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Code review in GovPress
+last_reviewed_at: ""
+---
+
+
+GovPress follows [the same process for software development](https://playbook.dxw.com/tech/how-we-do-development/) as the Tech Team. However, developers in the Tech Team work on a small number of long-running projects, with relatively large teams. In GovPress we work on a very large number of small projects, with small teams, so we have a slightly different emphasis on how we write code.
```

Comment:
To make it easier to develop locally, best to omit the domain

```suggestion
GovPress follows [the same process for software development](/tech/how-we-do-development/) as the Tech Team. However, developers in the Tech Team work on a small number of long-running projects, with relatively large teams. In GovPress we work on a very large number of small projects, with small teams, so we have a slightly different emphasis on how we write code.
```

Reply:
Thanks, I should have checked all the copy & pasting!